packages/server/lib/hooks/connection/workable-post-connection.ts

```diff
@@ -0,0 +1,27 @@
+import type { InternalNango as Nango } from './post-connection.js';
+import { isAxiosError } from 'axios';
+
+export default async function execute(nango: Nango) {
+    const connection = await nango.getConnection();
+
+    // rely on the customer provided subdomain if possible
+    if (connection.connection_config?.['subdomain']) {
+        return;
+    }
+
+    const response = await nango.proxy({
```

Comment:
Can you type (even lightly)?  e.g: `nango.proxy<WorkableAccounts>(`

Reply:
Fixed in 3d41db42